src/webapp/README.md

```diff
@@ -0,0 +1,117 @@
+| **Name**                     | `python-webapp`                                                              | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**  | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**            | **Web**                                                                      | Required value for this sample. <br/> Enables the required and optional settings for the app type. |
+| **Redirect URI**             | `http://localhost:5000/auth/redirect`                                        | Required value for this sample.                                                                    |
+| **Client secret**            | _Value shown in Azure portal_                                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **Implicit grant & hybrid flows** | _None selected_                                                         | This sample does not use the implicit grant or hybrid flows.                                       |
+| **Allow public client flows** | **No**                                                                      | This sample does not use a public client flow.                                                     |
+| **Token configuration**      | _No additional claims_                                                       | This sample does not rely on any additional claims existing in the tokens.                         |
+| **App roles**                | _Add an App role called `admin` for use by **Users/Groups**_                 | Required value for this sample. <bn /> One route requires your user to be assigned this role.      |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+Use these settings in your Enterprise Application for this sample app
+
+| Enterprise Application <br/> setting | Value for this sample app                                            | Notes                                                                                              |
+|:------------------------------------:|:---------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Users and groups**                 | _Add yourself with the **Role** of **admin**_                        | Required value for this sample. <bn /> One route requires your user to be assigned this role.      |
```

Comment:
wondering whether it is now settled and we should incorporate this as well in the `csharp` samples.

Reply:
Not sure.  I honestly didn't look at how the csharp one was set up.  I should have cross-referenced it.  I'll leave this comment open as we get more feedback on this.

Reply:
This would be the first time we direct anyone to use Enterprise Apps for adding a role and a user to that role, but that might not be a bad thing. Not sure on this, I'll need to run through it.

Reply:
Cool, let me know if there is a better way to do this.

Reply:
Nope, this is the right way to go - ship it.